--- Install_cernlib	2019-01-21 09:43:58.000000000 -0600
+++ Install_cernlib	2020-09-01 15:19:13.517283546 -0500
@@ -27,7 +27,7 @@
       a = 0
       end
 EOF
-GCCVSN=`cpp -dM comptest.F | grep __VERSION__ | cut -d" " -f3 | cut -c 2`
+GCCVSN=`cpp -dM comptest.F | grep __VERSION__ | cut -d" " -f3 | sed -s "s/\"//g" | cut -d . -f 1`
 FC=" "
 [ "$GCCVSN" = "3" ]&&FC=g77
 [ "$GCCVSN" -ge "4" ]&&FC=gfortran
--- Install_cernlib_and_lapack	2019-01-21 09:43:58.000000000 -0600
+++ Install_cernlib_and_lapack	2020-09-01 15:18:41.436517034 -0500
@@ -27,7 +27,7 @@
       a = 0
       end
 EOF
-GCCVSN=`cpp -dM comptest.F | grep __VERSION__ | cut -d" " -f3 | cut -c 2`
+GCCVSN=`cpp -dM comptest.F | grep __VERSION__ | cut -d" " -f3 | sed -s "s/\"//g" | cut -d . -f 1`
 FC=" "
 [ "$GCCVSN" = "3" ]&&FC=g77
 [ "$GCCVSN" -ge "4" ]&&FC=gfortran
